sdk/core/Azure.Core/src/Request.cs

```diff
@@ -2,48 +2,128 @@
 // Licensed under the MIT License.
 
 using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
-using System.Net.Http;
 using Azure.Core.Pipeline;
 
 namespace Azure.Core
 {
     /// <summary>
-    /// Represents an HTTP request. Use <see cref="HttpPipeline.CreateMessage()"/> or <see cref="HttpPipeline.CreateRequest"/> to create an instance.
+    /// Represents an HTTP request. Use <see cref="HttpPipeline.CreateMessage()"/>
+    /// or <see cref="HttpPipeline.CreateRequest"/> to create an instance.
     /// </summary>
 #pragma warning disable AZC0012 // Avoid single word type names
-    public abstract class Request : IDisposable
+    public abstract class Request : PipelineRequest
 #pragma warning restore AZC0012 // Avoid single word type names
     {
-        private RequestUriBuilder? _uri;
+        private RequestUriBuilder? _uriBuilder;
+        private string _method = RequestMethod.Get.Method;
+        private RequestContent? _content;
```

Comment:
can we reuse the existing field on the base class?

Reply:
I believe Azure.Core's `Request` already has a RequestContent field backing the virtual property -- this just makes it explicit.  You can see the original file in main here: https://github.com/Azure/azure-sdk-for-net/blob/9e57c09497c36f8ae981d93c4343fec56c22bf8d/sdk/core/Azure.Core/src/Request.cs#L45